### PR TITLE
fix(grafana): Update Prometheus dashboards

### PR DESCRIPTION
**Pull request by kskewes**
Recently Prometheus metric names were updated. This PR updates dashboards to use new naming scheme.

### CODE REVIEW

File: spinnaker-monitoring-third-party/third_party/prometheus/clouddriver-microservice-dashboard.json [hunk 1]
```diff
@@ -1034,9 +1034,9 @@
         "includeAll": true,
         "label": null,
         "multi": false,
-        "name": "Account",
+        "name": "Account_total",
```
Comment (jfillo) [inline spinnaker-monitoring-third-party/third_party/prometheus/clouddriver-microservice-dashboard.json]: Should stay "Account"

File: spinnaker-monitoring-third-party/third_party/prometheus/front50-microservice-dashboard.json [hunk 2]
```diff
@@ -769,15 +769,15 @@
...
+              "expr": "sum($Function(front50:storageServiceSupport:numAdded_total{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
               "intervalFactor": 2,
               "legendFormat": "add {{objectType}}",
               "metric": "",
               "refId": "A",
               "step": 40
             },
             {
-              "expr": "-1 * sum($Function(front50:storageServiceSupport:numRemoved{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
+              "expr": "-1 * sum($Function(front50:storageServiceSupport:numRemoved_total{instance=~\"$Instance\"}[$SamplePeriod])) by (objectType)",
```
Comment (jfillo) [inline spinnaker-monitoring-third-party/third_party/prometheus/front50-microservice-dashboard.json]: missing an update to `numUpdated` in this file

File: spinnaker-monitoring-third-party/third_party/prometheus/kubernetes-platform-dashboard.json [hunk 3]
```diff
@@ -1107,9 +1107,9 @@
         "includeAll": true,
         "label": "",
         "multi": false,
-        "name": "KubernetesAccount",
+        "name": "KubernetesAccount_total",
```
Comment (jfillo) [inline spinnaker-monitoring-third-party/third_party/prometheus/kubernetes-platform-dashboard.json]: Should remain "KubernetesAccount"

File: spinnaker-monitoring-third-party/third_party/prometheus/front50-microservice-dashboard.json (hunk 2 above)
Comment (kskewes) [reply spinnaker-monitoring-third-party/third_party/prometheus/front50-microservice-dashboard.json]: Thanks. Fixed.

File: spinnaker-monitoring-third-party/third_party/prometheus/clouddriver-microservice-dashboard.json (hunk 1 above)
Comment (kskewes) [reply spinnaker-monitoring-third-party/third_party/prometheus/clouddriver-microservice-dashboard.json]: Thanks. Fixed.

File: spinnaker-monitoring-third-party/third_party/prometheus/kubernetes-platform-dashboard.json (hunk 3 above)
Comment (kskewes) [reply spinnaker-monitoring-third-party/third_party/prometheus/kubernetes-platform-dashboard.json]: Thanks. Fixed.

File: spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json [hunk 4]
```diff
@@ -407,7 +407,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum($Function(orca:stage:invocations{isComplete=\"false\"}[$SamplePeriod])) by (instance, executionType, cloudProvider)",
+              "expr": "sum($Function(orca:stage:invocations_total{isComplete=\"false\"}[$SamplePeriod])) by (instance, executionType, cloudProvider)",
```
Comment (jfillo) [inline spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json]: This is probably a separate issue, but it doesn't look like the attribute `isComplete` is included in this metric. Causing this panel to never provide any data points

**Comment by jfillo** [comment]
I was able to verify all the dashboards except for `aws-platform`, `google-platform`, and `gate-microservice`

**Comment by ewiseblatt** [comment]
Thanks guys. Before I ahve a look, can you squash it all so I can commit when done?

File: spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json (hunk 4 above)
Comment (kskewes) [reply spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json]: There are a few like that. We could remove the `isComplete` label from the query to match the metric, like the below sample: 
```
orca:stage:invocations_total{application="myapp",cloudProvider="kubernetes",endpoint="monitoring",exported_instance="spin-orca-6f9b77fc4d-kcwj5:8083",exported_job="orca",instance="172.30.83.215:8008",job="spin-monitoring",namespace="spinnaker",pod="spin-orca-6f9b77fc4d-kcwj5",service="spin-monitoring",type="deployManifest"}
orca:stage:invocations_total{application="myapp",endpoint="monitoring",exported_instance="spin-orca-6f9b77fc4d-kcwj5:8083",exported_job="orca",instance="172.30.83.215:8008",job="spin-monitoring",namespace="spinnaker",pod="spin-orca-6f9b77fc4d-kcwj5",service="spin-monitoring",type="jenkins"}
```

Comment (kskewes) [reply spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json]: @ewiseblatt What do you think about this approach above?
Not sure if the `isComplete` label has been missed out or removed on purpose at some stage.
Happy to send additional commits and get them in this PR if desired.

Comment (ewiseblatt) [reply spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json]: The dashboard was written for the original orca. The rewrite earlier this year changed some things. Some metrics are the same and some changed, such as this tag no longer being there.

I'll followup offline with you and changes I started to make but had to put on hold before I could bring it to closure. Perhaps you can take it over the finish line.

Comment (kskewes) [reply spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json]: Sure. Thanks very much.

**Pull request merged**